cluster-autoscaler/cloudprovider/gce/gce_manager.go

```diff
@@ -1205,37 +1157,12 @@ func (m *gceManagerImpl) findMigsInRegion(region string, name *regexp.Regexp) ([
 	return links, nil
 }
 
-func (m *gceManagerImpl) findMigsInZone(zone string, name *regexp.Regexp) ([]string, error) {
-	filter := fmt.Sprintf("name eq %s", name)
-	links := make([]string, 0)
-	req := m.gceService.InstanceGroups.List(m.getProjectId(), zone).Filter(filter)
-	if err := req.Pages(context.TODO(), func(page *gce.InstanceGroupList) error {
-		for _, ig := range page.Items {
-			links = append(links, ig.SelfLink)
-			glog.V(3).Infof("autodiscovered managed instance group %s using regexp %s", ig.Name, name)
-		}
-		return nil
-	}); err != nil {
-		return nil, fmt.Errorf("cannot list managed instance groups: %v", err)
-	}
-	return links, nil
-}
-
 func (m *gceManagerImpl) getMigTemplate(mig *Mig) (*gce.InstanceTemplate, error) {
-	igm, err := m.gceService.InstanceGroupManagers.Get(mig.Project, mig.Zone, mig.Name).Do()
-	if err != nil {
-		return nil, err
-	}
-	templateUrl, err := url.Parse(igm.InstanceTemplate)
-	if err != nil {
-		return nil, err
-	}
-	_, templateName := path.Split(templateUrl.EscapedPath())
-	instanceTemplate, err := m.gceService.InstanceTemplates.Get(mig.Project, templateName).Do()
+	template, err := m.GceService.FetchMigTemplate(mig.GceRef)
```

Comment:
Do we need this method anymore? All it does is proxy to FetchMigTemplate, which is available everywhere where we can call this. Maybe we should just update the places calling it and scrap this method? Same for getZones.